<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_day</class>
 <widget class="QWidget" name="w_day">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>53</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>362</width>
    <height>53</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>362</width>
    <height>53</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="l_title">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>10</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>121</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>121</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>воскресенье</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_temp">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>61</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>61</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-55</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_type">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>149</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>149</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>149</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Пасмурно</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
